src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -336,6 +336,31 @@ internal Authority GetAuthority(IAccount account)
             return await handler.RunAsync(CancellationToken.None).ConfigureAwait(false);
         }
 
+        internal async Task<AuthenticationResult> ExchangeRefreshTokenAsync(string userProvidedRefreshToken)
+        {
+            var reqParams = new AuthenticationRequestParameters
+            {
+                SliceParameters = SliceParameters,
+                Authority = Instance.Authority.CreateAuthority(ServiceBundle, Authority, false),
+                ClientId = ClientId,
+                TokenCache = UserTokenCache,
+                Scope = new SortedSet<string>(),
+                RedirectUri = new Uri(RedirectUri),
+                RequestContext = CreateRequestContext(Guid.Empty),
```

Comment:
Nit: CreateRequestContext can be made private and the param can be removed (as it is always set to Guid.Empy)

Reply:
How about getting rid of it all together?

Reply:
As you wish, it's a tiny method, but keeping it might make the code just a little bit clearer.